.gitattributes

```diff
@@ -0,0 +1 @@
+*.js text eol=lf
```

Comment:
This is a workaround for https://github.com/MozillaFoundation/javascript-style-guide/issues/13.